packages/keyring-eth-ledger-bridge/test/document.shim.ts

```diff
@@ -1,24 +1,31 @@
-// eslint-disable-next-line import/no-mutable-exports
+// eslint-disable-next-line import-x/no-mutable-exports
```

Comment:
I'm not familiar with this one, how is it different from `import/no-mutable-exports`?

Reply:
Looks like the plugin got renamed:

- https://github.com/un-ts/eslint-plugin-import-x/issues/24#issuecomment-1991605123